### ISSUE TITLE
Bug: wired connection is shown as "Ping: none"

### ISSUE THREAD

**Issue by bersbersbers** [EFLFE/PingoMeter#3]
This is my network setup as seen from `tracert`:

```
>tracert 8.8.8.8

Tracing route to 8.8.8.8 over a maximum of 30 hops

  1     1 ms    <1 ms    <1 ms  10.0.0.6
  2     4 ms     2 ms     2 ms  10.0.0.1
  3    10 ms    10 ms     9 ms  xxx
  4    11 ms    11 ms    11 ms  xxx
  5    21 ms    21 ms    19 ms  xxx
  6    20 ms    20 ms    20 ms  xxx
  7    21 ms    22 ms    20 ms  172.253.64.119
  8    20 ms    20 ms    19 ms  8.8.8.8
```

So I have three PingoMeters running for `10.0.0.6`, `10.0.0.1`, and `8.8.8.8`, where for first one is shown as "Ping: none" intermittently.
![image](https://user-images.githubusercontent.com/12128514/87667116-14f0af00-c76a-11ea-9ee6-496afcf59be1.png)

Could it be that the `10.0.0.6` one is too fast and is shown as "Ping: none" incorrectly? I am referring to this piece of code:

https://github.com/EFLFE/PingoMeter/blob/5bdee3b4f9f77719be690762c92e9f61e44ce61f/NotificationIcon.cs#L172-L174

https://github.com/EFLFE/PingoMeter/blob/5bdee3b4f9f77719be690762c92e9f61e44ce61f/NotificationIcon.cs#L83-L94

It seems to me like `value == 0L` is not necessarily an exception.

**Comment by bersbersbers** [comment]
Commenting out these lines fixes the issue:
https://github.com/EFLFE/PingoMeter/blob/5bdee3b4f9f77719be690762c92e9f61e44ce61f/NotificationIcon.cs#L90-L94

**Comment by EFLFE** [comment]
Thanks for the report. I will fix it soon.

**Comment by EFLFE** [comment]
I think the problem is solved?

**Comment by bersbersbers** [comment]
Yes! I had re-built the app myself with the proposed fix, but I also verified 0.9.5 now and it's working well. Thank you!
